samples/gltf_viewer.cpp

```diff
@@ -721,11 +764,15 @@ int main(int argc, char** argv) {
             }
 
             colorGradingUI(app);
-        });
 
-        // Leave FXAA enabled but we also enable MSAA for a nice result. The wireframe looks
-        // much better with MSAA enabled.
-        view->setSampleCount(4);
```

Comment:
This codeline became bogus after we added a checkbox for MSAA.  It did no harm because it got clobbered later in the execution floor.